.github/actions/label/build.js

```diff
@@ -128,12 +128,17 @@ const listJobs = async ({ github, core, run_id }) => {
   });
 
   if (jobs.length === 0) {
-    core.setFailed(`Empty jobs from dispatched workflow`);
-    return;
+    core.setFailed("Empty jobs from dispatched workflow", jobs);
+  } else if (jobs.length == 1 && jobs[0].name === "matrix") {
+    core.info("Waiting for matrix job to be completed ... ");
+    await sleep(5000);
```

Comment:
Or we could use 

https://github.com/Codex-/await-remote-run

To be sure action was not expected/queued so waiting time exceeds 5 seconds

Reply:
looks good but seems hard to integrate in our script since it's a wrapped action, the basic logic are similar https://github.com/Codex-/await-remote-run/blob/1e64aec4261993d50c3c22da19b925106d07338f/src/api.ts#L243